test/index.test.js

```diff
@@ -1,9 +1,195 @@
 'use strict';
 
 const assert = require('chai').assert;
+const EventEmitter = require('events').EventEmitter;
```

Comment:
So generally when you alphabetize, capital letters come first (so `EventEmitter` should be before `assert`). But it's not a big deal